### PR TITLE
Item DB Updates

### LINKED ISSUE #1586: Essence of Evil
right now it's for Example
`4908,Essence_of_Evil_STR1,Essence of Evil STR1,6,20,,10,,,,,,,,,,,,,{ bonus bStr,1; bonus bBaseAtk,3; bonus bInt,-1;},{},{}`

and it must be
`4908,Essence_of_Evil_STR1,Essence of Evil STR1,6,20,,10,,,,,,,,1023,,,,,{ bonus bStr,1; bonus bBaseAtk,3; bonus bInt,-1;},{},{}`

the card place is 1023 
can put in any item type

the items id is : 4908 , 4909 , 4910 , 4911 , 4912 , 4913 , 4914 , 4915 , 4916 , 4917 , 4918 , 4919 , 4920 , 4921 , 4922 , 4923 , 4924 , 4925 .

in db/re/item_db.txt

i think the same in the sql files but not sure cuz i don't use them .

### PR DESCRIPTION
**Pull request by cydh**
- Corrected Aegis names of Enchancement Stones & Shadow Equipments, credits to http://divine-pride.net
- Corrected Price for Shadow equipments to 0.
- Fixed #1586 item position for Essence of Evils.

Signed-off-by: Cydh Ramdh cydh@pservero.com

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -11051,222 +11050,222 @@
...
+24027,S_Immune_Armor,Immune Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus2 bSubEle,0,1; },{},{}
+24028,S_Hard_Armor,Hard Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus bMaxHP,50; if(getrefine()>=7) { bonus bMaxHPrate,1; } },{},{}
+24029,S_Ancient_Armor,Ancient Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus bHit,(getrefine()>=7)?20:10; },{},{}
+24030,S_Critical_Armor,Critital Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus bCritical,(getrefine()>=7)?10:5; },{},{}
+24031,S_Kingbird_Weapon,King Bird Shadow Weapon,12,0,,0,,0,,0,0xFFFFFFFF,63,2,131072,,1,1,0,{ bonus bBaseAtk,10; if(getrefine()>=7) { bonus bLongAtkRate,1; } },{},{}
+24032,S_Cri_Hit_Weapon,Critical Hit Shadow Weapon,12,0,,0,,0,,0,0xFFFFFFFF,63,2,131072,,1,1,0,{ bonus bBaseAtk,10; if(getrefine()>=7) { bonus bCritAtkRate,1; } },{},{}
+24033,S_Healing_Weapon,Healing Shadow Weapon,12,0,,0,0:10,,,0,0xFFFFFFFF,63,2,131072,,1,1,,{ if (getrefine()>=7) bonus bHealPower2,5; },{},{}
+24034,S_Lucky_Weapon,Lucky Shadow Weapon,12,0,,0,,0,,0,0xFFFFFFFF,63,2,131072,,1,1,0,{ bonus bLuk,1; if(getrefine()>=7) { bonus bLuk,1; } if(getrefine()>=9) { bonus bLuk,1; } },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: the if refine >= 7 could also include the >= 9, because it cant be >= 9 without being >= 7

File: db/re/item_db.txt [hunk 2]
```diff
@@ -3012,88 +3012,87 @@
...
+4898,Matk4p,MATK+4%,6,10,,,,,,,,,,,,,,,{ bonus bMatkRate,4; },{},{}
+4899,Matk5p,MATK+5%,6,10,,,,,,,,,,,,,,,{ bonus bMatkRate,5; },{},{}
+4900,MHP5,MaxHP+5%,6,10,,,,,,,,,,,,,,,{ bonus bMaxHPrate,5; },{},{}
+4902,Def18,DEF+18,6,10,,,,,,,,,,,,,,,{ bonus bDef,18; },{},{}
+4903,Def21,DEF+21,6,10,,,,,,,,,,,,,,,{ bonus bDef,21; },{},{}
+4904,Atk6p,ATK+6%,6,10,,,,,,,,,,,,,,,{ bonus bAtkRate,6; },{},{}
+4905,Atk7p,ATK+7%,6,10,,,,,,,,,,,,,,,{ bonus bAtkRate,7; },{},{}
+4906,Matk6p,MATK+6,6,10,,,,,,,,,,,,,,,{ bonus bMatkRate,6; },{},{}
+4907,Matk7p,MATK+7%,6,10,,,,,,,,,,,,,,,{ bonus bMatkRate,7; },{},{}
+4908,Force1,Essence of Evil STR1,6,20,,10,,,,,0xFFFFFFFF,63,2,1021,,,,,{ bonus bStr,1; bonus bBaseAtk,3; bonus bInt,-1;},{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: Have you confirmed those as working? I think they might still be applied to a card slot like this.

File: db/re/item_db.txt [hunk 3]
```diff
@@ -11051,222 +11050,222 @@
...
+24026,S_Athena_Shield,Shadow Athena Shield,12,0,,0,,0,,0,0xFFFFFFFF,63,2,262144,,1,1,0,{ bonus bDef,(getrefine()>=7)?20:10; },{},{}
+24027,S_Immune_Armor,Immune Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus2 bSubEle,0,1; },{},{}
+24028,S_Hard_Armor,Hard Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus bMaxHP,50; if(getrefine()>=7) { bonus bMaxHPrate,1; } },{},{}
+24029,S_Ancient_Armor,Ancient Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus bHit,(getrefine()>=7)?20:10; },{},{}
+24030,S_Critical_Armor,Critital Shadow Armor,12,0,,0,,0,,0,0xFFFFFFFF,63,2,65536,,1,1,0,{ bonus bCritical,(getrefine()>=7)?10:5; },{},{}
+24031,S_Kingbird_Weapon,King Bird Shadow Weapon,12,0,,0,,0,,0,0xFFFFFFFF,63,2,131072,,1,1,0,{ bonus bBaseAtk,10; if(getrefine()>=7) { bonus bLongAtkRate,1; } },{},{}
+24032,S_Cri_Hit_Weapon,Critical Hit Shadow Weapon,12,0,,0,,0,,0,0xFFFFFFFF,63,2,131072,,1,1,0,{ bonus bBaseAtk,10; if(getrefine()>=7) { bonus bCritAtkRate,1; } },{},{}
+24033,S_Healing_Weapon,Healing Shadow Weapon,12,0,,0,0:10,,,0,0xFFFFFFFF,63,2,131072,,1,1,,{ if (getrefine()>=7) bonus bHealPower2,5; },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: I don't see any changes in the script. Why does github highlight this? You did nothing wrong here O_o

File: db/re/item_db.txt (hunk 2 above)
Comment (cydh) [inline db/re/item_db.txt]: Like I said before, when tested on Aegis server (Rayn's), it goes to card slot when u double click it. (But, who knows for recent Aegis).
I didn't find any iRO video.

@euphyy ever mentioned that Enhancement Stones has `compositionType` is 2, card is 1.
iirc, in Aegis scripts, this value is used to check if the compounded item is card or en.stone. Ours, just `if (item_id >= 4700) { is_enstone }`

**Comment by cydh** [comment]
i'll update the iRO Engrish name

File: db/re/item_db.txt (hunk 1 above)
Comment (cydh) [inline db/re/item_db.txt]: if refine +7, luk +1
if refine +9, luk +2 (or refine +7 luk +1, refine +9 luk +1)

**Pull request merged**